### PR TITLE
updated feature #working dir

### PR DESCRIPTION
**Pull request by tmsundaram**
Updated script to work in git based env. Added descriptive Inofrmation about script usage.

### CODE REVIEW

**Review by tmsundaram — COMMENTED** [review]
looks good

**Comment by tmsundaram** [comment]
lets merge to master

File: trading_fetch_rate/fetch_rate.sh [hunk 1]
```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
-
+#################################################################
+##Purpose: Fetch market price of given material and sent alert ##
+##Author : TM Sundaram                                         ##
+##Date   : 2020-05-09                                          ##
+##Version: 1.0.0                                               ##
+##Test   : Tested on Ubuntu 18.04                              ##
+#################################################################
```
Comment (tab-author) [inline trading_fetch_rate/fetch_rate.sh:8]: good

**Review by tab-author — APPROVED** [review]
time to merge

**Pull request merged**